recipes/CommonVoice/quantization/hparams/train_with_wav2vec.yaml

```diff
@@ -0,0 +1,60 @@
+################################
+# Recipe for Training K-Means Clustering on LibriSpeech Data
+# Using Self-Supervised Model-Based Representations
+#
+# It is used for creating discrete audio representations from LibriSpeech data.
+#
+# Author: Pooneh Mousavi (2023)
+################################
+# Seed needs to be set at top of yaml, before objects with parameters are made
+seed: 1986
+__set_seed: !apply:torch.manual_seed [!ref <seed>]
+output_folder: !ref results/CommonVoice/clustering/wav2vec/<seed>
+save_folder: !ref <output_folder>/save
+
+# Data files
+data_folder: !PLACEHOLDER  # e.g, /localscratch/cv-corpus-5.1-2020-06-22/fr
+train_tsv_file: !ref <data_folder>/train.tsv  # Standard CommonVoice .tsv files
+accented_letters: False
+language: en # use 'it' for Italian, 'rw' for Kinyarwanda, 'en' for english
+train_csv: !ref <save_folder>/train.csv
+skip_prep: False # Skip data preparation
+sample_rate: 16000
+
+# We remove utterance slonger than 10s in the train/dev/test sets as
+# longer sentences certainly correspond to "open microphones".
+avoid_if_longer_than: 10.0
+
+ssl_hub: facebook/wav2vec2-large-960h-lv60-self
+freeze_feature_extractor: True
+freeze_ssl: True
+ssl_folder: !ref <save_folder>/wav2vec_checkpoint
+ssl_layer_num: 7
+batch_size: 64 # batch_size for loading and extracting features. It is different from kmeans_batch_size.
+
+sorting: ascending
+
+# Dataloader options
+train_dataloader_opts:
+   batch_size: !ref <batch_size>
+
+ssl_model: !new:speechbrain.lobes.models.huggingface_transformers.wav2vec2.Wav2Vec2
+   source: !ref <ssl_hub>
+   output_norm: True
```

Comment:
`ouput_norm`  have to be set to False otherwise the Kmeans model will not converge